src/launchpad/size/insights/common.py

```diff
@@ -71,3 +75,25 @@ def generate(self, input: InsightsInput) -> LargeImageFileInsightResult:
         total_savings = sum(file.size // 2 for file in large_files)
 
         return LargeImageFileInsightResult(files=large_files, total_savings=total_savings)
+
+
+class LargeVideoFileInsight(Insight[LargeVideoFileInsightResult]):
+    """Insight for identifying video files larger than 10MB."""
+
+    def generate(self, input: InsightsInput) -> LargeVideoFileInsightResult:
+        size_threshold_bytes = 10 * 1024 * 1024  # 10MB
```

Comment:
Assuming just arbitrary right now? Fine with this number, might be good to just leave a comment that this was chosen arbitrarily.